SU-HSDO/suhumsci
# SHS-5544: Link in Footer when the theme is Colorful:Ocean and local footer is Dark does not meet color WCAG AA minimum contrast requirements

# READY FOR REVIEW

## Summary
Fix link in Footer when the theme is Colorful:Ocean and the local footer is dark to meet color WCAG AA minimum contrast requirements

## Need Review By (Date)
03/20

## Urgency
medium

## Steps to Test
1. Go to this site https://archaeology.suhumsci.loc/ or any site using the theme Coloful
2. Confirm the site is using the `Ocean` color pairing and the Footer is the `Dark`
3. Ensure now the more-link in the Footer is using the correct colors:
   - Link default color -	Tertiary-reversed (#00D5F5)
   - Link hover color -Tertiary-highlight
4. Confirm the other color pairings are still using the same colors they were using before

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
